src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -118,9 +119,7 @@ public void ConnectionWorkflow_ConnectionStep_ConnectedServerCanBeGetAndSet()
         {
             // Arrange
             var connectionInfo = new ConnectionInformation(new Uri("http://server"));
-#pragma warning disable IDE0017 // Simplify object initialization
-            ConnectionWorkflow testSubject = new ConnectionWorkflow(serviceProvider, host, new RelayCommand(() => { }));
-#pragma warning restore IDE0017 // Simplify object initialization
+            ConnectionWorkflow testSubject = CreateTestSubject(serviceProvider, host, new RelayCommand(() => { }));
```

Comment:
Changing the tests to use a `CreateTestSubject` helper made it easier to pass a dummy `IThreadHandling` instance to the constructor.